index.js

```diff
@@ -138,7 +148,11 @@ process.stdin.on('readable', () => {
 process.stdin.on('end', () => {
     const moments = parseDatesFromArgv();
 
-    process.stdout.write(`Users that contributed between ${moments[0]} and ${moments[1]} \n`);
+    process.stdout.write(
```

Comment:
This line is getting long and complicated enough that I would extract local consts for the two injected values.

Reply:
After storing the values as datetime objects instead of strings, the line became short again.